server/app/api/v1/endpoints/twilio_webhooks.py

```
@@ -0,0 +1,214 @@
+from app.services.customer_db_service import CustomerDBService
+from app.services.twilio_sms_service import get_twilio_sms_service
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.post("/status-webhook")
+async def twilio_status_webhook(
+    request: Request,
+    db: AsyncSession = Depends(get_db)
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)